factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/registry/transformation/cache/PgTransformationCache.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2017-2020 factcast.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.factcast.store.pgsql.registry.transformation.cache;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.factcast.core.Fact;
+import org.joda.time.DateTime;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import liquibase.integration.spring.SpringLiquibase;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class PgTransformationCache implements TransformationCache {
+    private final JdbcTemplate jdbcTemplate;
+
+    // note that SpringLiquibase needs to be injected in order to make sure it
+    // is initialized before we're intentionally not using @DependsOn here,
+    // as a change of the beanname within liquibase would break our code
+    private final SpringLiquibase liquibase;
```

Comment:
please create constructor and @autowire(required=false) liquibase

Reply:
Mhh....`PgTransformationCache` is no bean/Component so I think theres no need for @autowired !?  🤔

Reply:
if it is not, then why is SpringLiquibase a dependency, anyway?